gameplay-editor/src/gameplay.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#ifdef SDL_VIDEO_DRIVER_X11
+#include <QtCore>
```

Comment:
Also I don't want to have any Qt in the gameplay.h
Only the gameplay-editor should require Qt.